tests/TestCase/Http/ServerRequestTest.php

```diff
@@ -1550,10 +1550,10 @@ public function testAllowMethod(): void
             'REQUEST_METHOD' => 'PUT',
         ]]);
 
-        $this->assertTrue($request->allowMethod('put'));
+        $request->allowMethod('put');
 
         $request = $request->withEnv('REQUEST_METHOD', 'DELETE');
-        $this->assertTrue($request->allowMethod(['post', 'delete']));
+        $request->allowMethod(['post', 'delete']);
```

Comment:
You should change this to assertNull() instead, else this test method doesn't perform any assertions and also codecov complains about no test coverage for the return call.

Reply:
Checking void methods with asserts sounds wrong.
Shouldnt

     #[DoesNotPerformAssertions] 

suffice?

Reply:
Jep, seems like codecov is smart enough to see that void methods dont need the coverage then.
All green.